e2e-tests/cucumber-html-report.js

```diff
@@ -0,0 +1,27 @@
+const report = require("multiple-cucumber-html-reporter");
+
+report.generate({
+  jsonDir: "./cypress/cucumber-json",
+  reportPath: "./reports/cucumber-html-report.html",
+  metadata: {
```

Comment:
@madhu-355 I think metadata (e.g. browser device, platform) should be set from the environment and not hardcoded.